docs/bare_metal/c_small_libs.md

```diff
@@ -62,6 +64,29 @@ After you have completed the steps above, add `small` to the `supported_c_libs`
 }
 ```
 
+### Non-returning main()
+
+Arm microlib doesn't support exiting from `main()`; attempting to exit from `main()` causes a bare metal application to crash. Here we show two ways to prevent this.
```

Comment:
So we say both "exit" and "return" on this page. We should stick to one.

Reply:
I will use "return"